vncviewer/OptionsDialog.h

```diff
@@ -125,7 +129,12 @@ class OptionsDialog : public Fl_Window {
   Fl_Int_Input *desktopHeightInput;
   Fl_Check_Button *remoteResizeCheckbox;
   Fl_Check_Button *fullScreenCheckbox;
-  Fl_Check_Button *fullScreenAllMonitorsCheckbox;
+
+  Fl_Group *fullScreenModeGroup;
```

Comment:
Do we need to keep track of this group?

Reply:
Previously, it was needed because I disabled that group when another option was chosen for fullscreen. This has been changed, so it can be removed.